section3/reflection.md

```diff
@@ -2,12 +2,30 @@
 
 1. In Section 3, we removed even more supports and pushed you to learn a bit more independently, read through technical documentation, and use your resources more. How did that feel?
 
-1. Think of a web or mobile application you use regularly. Where is an example of a place that the developers probably use Arrays? Explain.
+- It was hard to understand what you asked. By watching videos and asking to friends, I tried to do something. While searching , I have found different ways how to write my codes.But I am still spending more time until figure out the problems by myself.
```

Comment:
This is a lot of what my day to day job looks like, so you're getting great practice 🙂